api/SubmitJob/cloudformation.yml

```diff
@@ -0,0 +1,63 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Outputs:
+
+  LambdaArn:
+    Value: !GetAtt Lambda.Arn
+
+Parameters:
+  JobQueueArn:
+    Type: String
+
+  JobDefinition:
```

Comment:
this should probably be `JobDefinitionArn` to be specific

Reply:
I don't think it is the actual Arn, it is the same parameter name we used in the other one

Reply:
Yeah, that's kind of my point, we don't remember what the value is support to look like.  I thinking it's `arn:version`, actually.  Adding `Description`s for all our parameters might be in our future.

Reply:
I think I prefer the idea of adding descriptions